src/drops/ERC721Drop.sol

```diff
@@ -57,30 +60,30 @@ contract ERC721Drop is
   ERC721DropStorageV1
 {
   /// @dev This is the max mint batch size for the optimized ERC721A mint contract
-  uint256 internal immutable MAX_MINT_BATCH_SIZE = 8;
+  uint256 constant MAX_MINT_BATCH_SIZE = 8;
```

Comment:
what's the reason for changing `internal immutable` to `constant`? constructor vs initialize?